x-pack/solutions/security/packages/features/src/security/v2_features/kibana_features.ts

```diff
@@ -74,6 +91,21 @@ export const getSecurityV2BaseKibanaFeature = ({
   ),
   privileges: {
     all: {
+      replacedBy: {
+        default: [
+          { feature: TIMELINE_FEATURE_ID, privileges: ['all'] },
+          { feature: NOTES_FEATURE_ID, privileges: ['all'] },
```

Comment:
I don't think we need to add these replacements for the `V2` -> `V3`. 
`V2` should already have the  _timeline_ and _notes_ features correctly configured, that's only needed for the V1 -> V2 migration.

Reply:
whoopsies - you are right! even worse, these lines would introduce an erroneous migration by enabling notes/timelines for users who haven't had those privileges so far.

573322ba7231c9f960525f2873ada72530b262d5

nice catch, thanks!